### ISSUE TITLE
Background celery runners

### ISSUE THREAD

**Issue by P-T-I** [cve-search/CveXplore#278]
Following up on the #271 issue; I'm seeing several tasks that could be delegated to a background task. Worked with celery quite a lot, would suffice for this purpose.

- [ ] Setup celery daemon
- [ ] Create docker (compose) files
- [ ] Add redis redbeat for dynamic task updating
- [ ] Create cli functions for task creation, status checks etc
- [ ] Create tasks for auto update
- [ ] Create tasks for reprocessing (certain) cve's on cpe update

**Comment by oh2fih** [comment]
Sounds good. Could you keep the docker optional? The tasks & scheduler could be run as SystemD services, which is also [supported by Celery](https://docs.celeryq.dev/en/stable/userguide/daemonizing.html).

**Comment by P-T-I** [comment]
Yeah sure, systemd is off course an option! I'll add that one as well!
